File: program_data/github_java_sort_function/7/122.java
  private static int[] sort(int[] sortMe, int shellSize) {
    if(shellSize <= 0) return new int[0];


    int temp;
    
    for (int i=shellSize; i<sortMe.length; i+=shellSize) {

      for (int j=i-shellSize; j>=0; j-=shellSize) {
     
        
       
      
        if( sortMe[j] > sortMe[j+shellSize] ) {
          temp = sortMe[j];
          sortMe[j] = sortMe[j+shellSize];
          sortMe[j+shellSize] = temp;
        }
      }
    }
    return sortMe;
  }
  

File: program_data/PKU_raw/104/122.c
void main()
{
	int x[100],y[100],i,j,p,q,k,temp=0;
	scanf("%d%d",&x[0],&y[0]);
	for(i=0;x[i]!=0;i++)
	{
		
		x[i+1]=x[i]/2;
	}
    for(p=0;y[p]!=0;p++)
	{
     
		y[p+1]=y[p]/2;
	}
	for(q=0;q<=i;q++)
	{
		for(j=0;j<=p;j++)
		{
			if(x[q]==y[j])
			{
				temp=1;
				k=q;
				break;
			}
		}
		if(temp==1)
			break;
	}
	printf("%d",x[k]);
}